pipeline/BuildBaseImage/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM nodesource/wheezy:4.4.6
+
+RUN apt-get update && apt-get install -y \
```

Comment:
What's the average failing rate , if without a pre-build based docker , in either US or SH ?
I'm thinking with this PR as a role model, the latter PR gate tasks will all have to create pre-build base docker/pipeline .